server/graphql/loaders/index.js

```diff
@@ -54,16 +54,40 @@ export const loaders = req => {
 
   // Collective - Balance
   context.loaders.Collective.balance = new DataLoader(ids =>
-    models.Transaction.findAll({
-      attributes: [
-        'CollectiveId',
-        [sequelize.fn('COALESCE', sequelize.fn('SUM', sequelize.col('netAmountInCollectiveCurrency')), 0), 'balance'],
-      ],
-      where: { CollectiveId: { [Op.in]: ids } },
-      group: ['CollectiveId'],
-    })
+    sequelize
+      .query(
+        `
+        WITH "blockedFunds" AS (
+          SELECT
+            e."CollectiveId", COALESCE(sum(e.amount), 0) as sum
+          FROM
+            "Expenses" e
+          WHERE
+            e."CollectiveId" IN (:ids)
+            AND (
+              e.status = 'SCHEDULED_FOR_PAYMENT'
+              OR (
+                e.status = 'PROCESSING' AND e.data ->> 'payout_batch_id' IS NOT NULL
```

Comment:
As per https://github.com/opencollective/opencollective-api/pull/3942#discussion_r433928267, I assume the reason for filtering on `payout_batch_id` is because for Transferwise, processing expenses already have transactions right?

Reply:
Yes, that's exactly right.
We have TransferWise expenses that are processing and are already accounted for in a transaction.